op_builder/inference_core_ops.py

```diff
@@ -60,13 +60,13 @@ def sources(self):
         sources = [
             "inference/v2/kernels/core_ops/core_ops.cpp",
             "inference/v2/kernels/core_ops/bias_activations/bias_activation.cpp",
-            "inference/v2/kernels/core_ops/bias_activations/bias_activation.cu",
+            "inference/v2/kernels/core_ops/bias_activations/bias_activation_cuda.cu",
```

Comment:
@jinzhen-lin - why rename these files to contain the _cuda suffix?

Reply:
Both `bias_activation.cpp` and `bias_activation.cu` generate `bias_activation.o`. Without renaming, ninja would throw an error
```
ninja: error: build.ninja:29: multiple rules generate xxxx/bias_activation.o
```

Reply:
I see - I know we previously had ninja support and that it is broken, but we should also get a review here from @mrwyattii

Reply:
I think this PR looks good. Since we don't have test workflow for XPU & HPU accelerators on this repo (yet), we should verify manually that we're not breaking anything there before merging.

Reply:
@delock, @nelyahu, @hipudding FYI

Please note this sensitive PR since we could only validate on cuda/hip/cpu.